onnxruntime/core/util/thread_utils.cc

```diff
@@ -87,6 +87,10 @@ CreateThreadPoolHelper(Env* env, OrtThreadPoolParams options) {
   }
   // override affinity setting if specified from customer
   if (!options.affinity_str.empty()) {
+#if defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)
+    ORT_THROW("Setting thread affinity is not implemented in this build.");
+    return nullptr;
+#else
     to.affinities = ReadThreadAffinityConfig(options.affinity_str);
```

Comment:
Is the implementation of ReadThreadAffinityConfig intended to be file scope? Not sure linker can throw it away as-is, so put in an anonymous namespace or make static if it's meant to be file scope.

Reply:
[now](https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=835135&view=logs&j=cbd9a98b-c201-5ad2-0e23-4c99a370d00f&t=adf49265-58d4-533c-580f-c52bffeeb92d)
[before](https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=832996&view=logs&j=cbd9a98b-c201-5ad2-0e23-4c99a370d00f&t=adf49265-58d4-533c-580f-c52bffeeb92d)

Reply:
Cool. Would be good to fix though to make it clear the function is intended to have file scope.

Reply:
Sure, done.